content/en/docs/ops/deployment/requirements/index.md

```diff
@@ -35,19 +41,41 @@ requirements:
+    are [enforced](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#enabling-pod-security-policies)
+    in your cluster and unless you use the [Istio CNI Plugin](/docs/setup/additional-setup/cni/), your pods must have the
+    `NET_ADMIN` and `NET_RAW` capabilities allowed. The initialization containers of the Envoy
+    proxies require these capabilities.
+
+    To check if the `NET_ADMIN` and `NET_RAW` capabilities are allowed for your pods, you need to check if their
+    [service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/)
+    can use a pod security policy that allows the `NET_ADMIN` and `NET_RAW` capabilities.
+    If you haven't specified a service account in your pods' deployment, the pods run using
+    the `default` service account in their deployment's namespace.
+
+    To list the capabilities for a service account, replace `<your namespace>` and `<your service account>`
+    with your values in the following command:
+
+    {{< text bash >}}
+    $ for psp in $(kubectl get psp -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}"); do if [ $(kubectl auth can-i use psp/$psp --as=system:serviceaccount:<your namespace>:<your service account>) = yes ]; then kubectl get psp/$psp --no-headers -o=custom-columns=NAME:.metadata.name,CAPS:.spec.allowedCapabilities; fi; done
+    {{< /text >}}
+
+    For example, to check for the `default` service account in the `default` namespace, run the following command:
+
+    {{< text bash >}}
+    $ for psp in $(kubectl get psp -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}"); do if [ $(kubectl auth can-i use psp/$psp --as=system:serviceaccount:default:default) = yes ]; then kubectl get psp/$psp --no-headers -o=custom-columns=NAME:.metadata.name,CAPS:.spec.allowedCapabilities; fi; done
+    {{< /text >}}
+
+    If you see `NET_ADMIN` and `NET_RAW` or `*` in the list of capabilities of one of the allowed
+    policies for your service account, your pods have permission to run the Istio init containers.
+    Otherwise, you will need to [provide the permission](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#authorizing-policies).
 
-- **Deployments with app and version labels**: We recommend adding an explicit
-  `app` label and `version` label to deployments. Add the labels  to the
-  deployment specification of pods deployed using the Kubernetes `Deployment`.
-  The `app` and `version` labels add contextual information to the metrics and
-  telemetry Istio collects.
+- **Pods with app and version labels**: We recommend adding an explicit
+  `app` label and `version` label to the specification of the pods deployed using
+  a Kubernetes `Deployment`. The `app` and `version` labels add contextual information
+  to the metrics and telemetry that Istio collects.
 
-    - The `app` label: Each deployment specification should have a distinct
+    - The `app` label: Each deployment should have a distinct
```

Comment:
@douglas-reid is this still best practice or did we switch to allowing/recommended the other k8s app label?

Reply:
the PR making the switch never was merged and we never switched to recommending as far as i can recall. However, we support the other label for canonical service derivation. I can't recall if our dashboards use "app" anywhere.

Reply:
OK, I'll leave it as is in this PR. If it needs updating, we'll address in a future PR.